src/Tools/dotnet-symbol/Program.cs

```diff
@@ -597,8 +599,7 @@ internal void VerifyCoreDump()
 
         private IEnumerable<string> GetInputFiles()
         {
-            IEnumerable<string> inputFiles = InputFilePaths.SelectMany((string file) =>
-            {
+            IEnumerable<string> inputFiles = InputFilePaths.SelectMany((string file) => {
```

Comment:
Seems to be the desired formatting in the project. Auto-format on saved changed it.